### DISCUSSION TITLE
q-select: default style and q-scroll-area scrollbar

### DISCUSSION THREAD

**Discussion by amirny2205** [quasarframework/quasar#15308] [General - Components / Directives / etc]
Hi! 
2 questions about q-select:
1) how do I change it's elments' inactive style? I didn't find such option
![style = ?](https://user-images.githubusercontent.com/105064582/213842090-997818c9-0188-4462-85ac-37b377b2d9ab.png)
2) Is there any way I can wrap it's options in a q-scroll-area, so that I have a different scrollbar? The native scrollbar looks ugly

**Comment by MilosPaunovic** [comment]
1. Using CSS
2. Using CSS

**Reply by amirny2205** [reply]
> 1. Using CSS
> 2. Using CSS

thanks! I have managed to solve the first problem,
The second problem might be resolved with https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp , though it cannot as there is no option for firefox. It doesn't work on firefox.

What I've noticed is that the scrollbar changes to a very decent looking one in adaptive mode, replacing every scrollbar not only the main one, even in firefox.
![decent looking scrollbar in adaptive](https://user-images.githubusercontent.com/105064582/214769581-cacc96f5-13a7-4c18-b7d4-95e55077cf68.png)
Is there any way I can force it in desktop? Thanks

**Reply by MilosPaunovic** [reply]
Create a Codepen reproduction of what you have so far.

**Reply by amirny2205** [reply]
> Create a Codepen reproduction of what you have so far.

as for the adaptive scrollbars, simply enter adaptive mode (ctrl+shift+m on firefox, button with a tablet and mobile in dveloper tools in chrome) and you'll see all the scrollbars change, even those which are in q-select, for example. It works here on github

In case to see that scrollbar settings(webkit) are not working on firefox, you can simply open this page and examples in firefox: [W3S How TO - Custom Scrollbar](https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp)

yes we could use firefox-specific settings for scrollbar,
```
{
  height: 100px;
  overflow-y: scroll;
  scrollbar-color: grey grey;
  scrollbar-width: thin;
}
```
though these are lacking: thumb border radius, background, shadow and width in pixels, probably somethings else
The fact is adaptive scolls have all that stuff and look pretty in firefox
